packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/cap_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+void main() {
```

Comment:
Just to make sure we don't accidentally regress anything during this transition: have you double-checked that all of these tests pass with *and without* your code changes?

Reply:
I've confirmed this is true.